rlmeta/agents/dqn/apex_dqn_agent.py

```diff
@@ -243,7 +266,7 @@ def __call__(self, index: int):
                             self._optimizer, self._batch_size, self._grad_clip,
                             self._multi_step, self._gamma,
                             self._learning_starts, self._sync_every_n_steps,
-                            self._push_every_n_steps, self._collate_fn)
+                            self._push_every_n_steps, self._collate_fn, additional_models_to_update=self._additional_models_to_update)
```

Comment:
Please try to format this file by yapf. We have the line limit of 80 characters.